packages/react-components/react-search/src/components/SearchBox/useSearchBox.tsx

```diff
@@ -16,12 +22,28 @@ import { DismissRegular, SearchRegular } from '@fluentui/react-icons';
 export const useSearchBox_unstable = (props: SearchBoxProps, ref: React.Ref<HTMLInputElement>): SearchBoxState => {
   const { size = 'medium', disabled = false, contentBefore, dismiss, contentAfter, ...inputProps } = props;
 
+  const rootRef = React.useRef<HTMLDivElement>(null);
```

Comment:
Possibly a better name for this is `searchBoxRootRef` since it's not the root ref for this component but rather the root of the root :)

Reply:
or `inputRootRef`, since it's the root of the Input, which is the root of the SearchBox?

root naming does get weird when it's nested like this :D